corehq/apps/smsbillables/async_handlers.py

```diff
@@ -143,9 +143,16 @@ def _directed_fee(direction, backend_api_id, backend_instance_id):
 
         for backend_instance in backends:
             backend_instance = backend_instance.wrap_correctly()
+            # Skip Testing backends
             if isinstance(backend_instance, TestSMSBackend):
                 continue
 
+            # skip if country is not in supported countries
+            if backend_instance.supported_countries:
+                supported_country_codes = [code for code in backend_instance.supported_countries]
```

Comment:
why is this step necessary - can't we just use backend_instance.supported_countries?